Header header
float64[] map_size
float64[] map_center
float64[] cell_size
NDTCellMsg[] cells
